exercism/diamond/diamond.js

```diff
@@ -0,0 +1,15 @@
+var Diamond = function() {}
+
+Diamond.prototype.makeDiamond = function(letter){
+  var alphabet = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z']
+
+var space = ' '
+var leadingSpace = function() {
+  alphabet.indexOf(letter) - index of that letter
+}
+var trailingSpace =
+var outsideSpace =
```

Comment:
If you intended to instantiate a variable with no initialized value, you simply write:
```
var trailingSpace
var outsideSpace
```